src/nih_wayfinding/app/scripts/views/navigate/report/report-controller.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function ReportController(
+        NavbarConfig, Reporting
```

Comment:
Be consistent, if only a few args, keep all on the line of the function definition